05-data-structures-part2.Rmd

```diff
@@ -137,14 +137,17 @@ gapminder <- read.csv("data/gapminder-FiveYearData.csv")
 
 > ## Miscellaneous Tips {.callout}
 >
-> 1. Another type of file you might encounter are tab-separated
-> format. To specify a tab as a separator, use `"\t"`.
+> 1. Another type of file you might encounter are tab-separated value files (.tsv). To specify a tab as a separator, use `"\\t"` or `read.delim()`.
 >
 > 2. You can also read in files from the Internet by replacing
-> the file paths with a web address.
+> the file paths with a web address. For example,
+> 
+> ```{r eval=FALSE, echo=TRUE}
+> gapminder <- read.csv("https://raw.githubusercontent.com/jrnold/r-novice-gapminder/gh-pages/data/gapminder-FiveYearData.csv")
```

Comment:
Can you change this url to point to the main repo
https://raw.githubusercontent.com/swcarpentry/r-novice-gapminder/gh-pages/data/gapminder-FiveYearData.csv